test/integration/generated/bodyComplex/src/operations/array.ts

```diff
@@ -69,7 +70,8 @@ export class Array {
 
   /**
    * Put complex types with array property which is empty
-   * @param complexBody Please put an array with 4 items: "1, 2, 3, 4", "", null, "&S#$(*Y", "The quick brown fox jumps over the lazy dog"
+   * @param complexBody Please put an array with 4 items: "1, 2, 3, 4", "", null, "&S#$(*Y", "The quick
+   * brown fox jumps over the lazy dog"
```

Comment:
I guess it'd be difficult to have the wrapper also indent subsequent lines to be equal to the starting position of the docstring in the first line

Reply:
We can pass it a param indentationType which we could use to decide what kind of indentation to have. For instance IndentationType.JSDocParam would be `"      "`